apps/services/user-notification/infra/user-notification.ts

```diff
@@ -16,6 +16,17 @@ export const userNotificationServiceSetup = (): ServiceBuilder<'user-notificatio
     })
     .liveness('/liveness')
     .readiness('/liveness')
+    .ingress({
+      primary: {
+        host: {
+          dev: 'user-notification-xrd',
```

Comment:
I am wondering about the naming convention. I assume xrd means X-Road.

Will this service be exposed through xroad?

Reply:
xrd stands for x-road, not sure where the naming comes from but we have another internal ingress targetted for x-road with a similar name

Reply:
For reference this was resolved on slack.